### PR TITLE
Updated documentation about the metadata service test

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS

@jframos 
### DESCRIPTION

Updated documentation about metadata service tests.

### CODE REVIEW

**Comment by jframos** [comment]
LGTM

**Comment by pratid** [comment]
Have you checked the Restructured Text markup? You could use [restview](https://mg.pov.lt/restview/) locally.

File: fiware-region-sanity-tests/README.rst [hunk 1]
```diff
@@ -241,21 +241,33 @@ rest of standard parsers bundled in NGSI Adapter package.
...
 to the *HTTP PhoneHome service* running in the configured HOST:PORT
 (*phonehome_endpoint* configuration). If this value is not set, this test will be skipped.
 
+There is another test implemented:
+
+* Test whether it is possible to deploy an instance and check that metadata service is working properly.
+
+This test is checking if openstack metadata service is working. It gets the metadata information about
+the VM deployed and return to the HTTP phonehome server.
+
```
Comment (pratid) [inline fiware-region-sanity-tests/README.rst:265]: what about: "and sends metadata back to the PhoneHome server to check whether is it correct"?

File: fiware-region-sanity-tests/README.rst [hunk 2]
```diff
@@ -241,21 +241,33 @@ rest of standard parsers bundled in NGSI Adapter package.
...
+Those test cases should return the information to the HTTP phonehome server and each one return that information
+to a different path.
```
Comment (pratid) [inline fiware-region-sanity-tests/README.rst:267]: Both test cases...

File: fiware-region-sanity-tests/README.rst [hunk 3]
```diff
@@ -272,11 +284,17 @@ The HTTP PhoneHome server waits for *POST HTTP requests* from VMs.
...
-When a request is received, HTTP PhoneHome server will inform, through the published object,
-to all connected tests about the event (broadcasting). This signal contains the
-hostname of the VM (the one received in the HTTP POST body). This signal will be take into account by
-tests that are waiting for a signal with this hostname value; the other tests will ignore it and will keep on
-listening new signals with the correct data (correct hostname) to them.
+When a request is received, HTTP PhoneHome server will inform all connected tests , through the published object,
+about the event (broadcasting). This signal contains the
+hostname of the VM (the one received in the HTTP POST body or the one received in the HTTP Header).
+Server is waiting in two different resources.
+- "/phonehome" to receive the hostname of the VM in the HTTP POST body.
```
Comment (pratid) [inline fiware-region-sanity-tests/README.rst]: I think a blank line is needed here...

File: fiware-region-sanity-tests/README.rst [hunk 4]
```diff
@@ -272,11 +284,17 @@ The HTTP PhoneHome server waits for *POST HTTP requests* from VMs.
...
 
-When a request is received, HTTP PhoneHome server will inform, through the published object,
-to all connected tests about the event (broadcasting). This signal contains the
-hostname of the VM (the one received in the HTTP POST body). This signal will be take into account by
-tests that are waiting for a signal with this hostname value; the other tests will ignore it and will keep on
-listening new signals with the correct data (correct hostname) to them.
+When a request is received, HTTP PhoneHome server will inform all connected tests , through the published object,
+about the event (broadcasting). This signal contains the
+hostname of the VM (the one received in the HTTP POST body or the one received in the HTTP Header).
+Server is waiting in two different resources.
```
Comment (pratid) [inline fiware-region-sanity-tests/README.rst]: I think a blank line is needed here...

**Comment by pratid** [comment]
LGTM

**Pull request merged**